### PR TITLE
PEP8 corrections/cleaning

### PR DESCRIPTION
**Pull request by Williangalvani**

### CODE REVIEW

File: src/protocol/templates/ping1d.py.in [hunk 1]
```diff
@@ -3,26 +3,31 @@
 # Ping1D.py
 # A device API for the Blue Robotics Ping1D echosounder
 
-#~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+# ~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
 # THIS IS AN AUTOGENERATED FILE
 # DO NOT EDIT
```
Comment (42B) [inline src/protocol/templates/ping1d.py.in]: Are you supposed to edit this file?

File: src/protocol/templates/pingmessage.py.in [hunk 2]
```diff
@@ -3,10 +3,10 @@
 # {{class_info.name}}.py
 # Python implementation of the Blue Robotics 'Ping' binary message protocol
 
-#~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
+# ~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!~!
 # THIS IS AN AUTOGENERATED FILE
 # DO NOT EDIT
```
Comment (42B) [inline src/protocol/templates/pingmessage.py.in]: Same here?

File: src/protocol/templates/ping1d.py.in (hunk 1 above)
Comment (Williangalvani) [reply src/protocol/templates/ping1d.py.in]: yes, this is the template generating the autogenerated file.

File: src/protocol/templates/pingmessage.py.in (hunk 2 above)
Comment (Williangalvani) [reply src/protocol/templates/pingmessage.py.in]: yes, this is the template generating the autogenerated file.

File: src/protocol/templates/ping1d.py.in (hunk 1 above)
Comment (42B) [reply src/protocol/templates/ping1d.py.in]: @patrickelectric 

Not at all, I just like reading code to try and learn from it.

Comment (patrickelectric) [reply src/protocol/templates/ping1d.py.in]: Sorry for asking, but there is some new bots in github and I was curious about your virtual existence, hehe.
Thank you for your inputs, and be welcome to be around.

Comment (42B) [reply src/protocol/templates/ping1d.py.in]: @patrickelectric 

No problem at all! I was thinking it was the short amount of time from when the PR was submitted to when I commented which caused you to think that but let me explain. I go to this [url](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+language%3Apython) and refresh the page to look at all the new pull requests for Python. It was purely random as to me coming across your repo today!

Comment (42B) [reply src/protocol/templates/ping1d.py.in]: @patrickelectric 

Nice to meet you too! See you around!

Comment (jaxxzer) [reply src/protocol/templates/ping1d.py.in]: @42B found us out, this text should not appear at the top of this file (the edit-able template), but should be inserted by the script that generates the result.

Comment (jaxxzer) [reply src/protocol/templates/ping1d.py.in]: https://github.com/bluerobotics/ping-protocol/issues/107

File: src/protocol/generate-python.py [hunk 3]
```diff
@@ -64,6 +64,7 @@ def is_vector(payload):
 allMessages.update(messageDict['general'])
 
 json_data['all_msgs'] = allMessages
+json_data['all_fields'] = list(set([msg['name'] for key, message in allMessages.items() for msg in message['payload']]))
```
Comment (jaxxzer) [inline src/protocol/generate-python.py]: This is a pretty complex statement. Please explain, or break it down for readability.

Comment (Williangalvani) [reply src/protocol/generate-python.py]: oops, I was going to add a comment explaining but forgot, my bad.

File: src/protocol/templates/pingmessage.py.in [hunk 4]
```diff
@@ -370,31 +376,32 @@ if __name__ == "__main__":
...
-    test_profile_buf = bytearray([0x42, 0x52,  0x1E, 0x00,  0x14, 0x05,  0x05, 0x06,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0x00,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0x00,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0x00,
-                               0x68, 0x65, 0x6c, 0x6c, 0x6f,
-                               0x1, 0x2, 0x63, 0x64,
-                               0xB2, 0x0A])
+    test_nack_buf = bytearray(
+        [0x42, 0x52, 0x06, 0x00, 0x03, 0x00, 0x05, 0x06, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x00, 0xBC, 0x02])
+    test_profile_buf = bytearray([0x42, 0x52, 0x1E, 0x00, 0x14, 0x05, 0x05, 0x06,
```
Comment (jaxxzer) [inline src/protocol/templates/pingmessage.py.in]: what happened here? removed newlines?

File: src/protocol/templates/pingmessage.py.in [hunk 5]
```diff
@@ -41,24 +42,27 @@ payloadDict = {
 {{structToken[field.type]}}
 {%- endif %}
 {% endfor %}",
-                       "field_names": (
+        "field_names": (
```
Comment (jaxxzer) [inline src/protocol/templates/pingmessage.py.in]: I do believe we should keep this the way that it is for readability: https://github.com/bluerobotics/ping-python/blob/master/Ping/PingMessage.py#L59

File: test/generate-json.py [hunk 6]
```diff
@@ -15,7 +15,7 @@
 # PingMessage is generated by this project
 import sys
 sys.path.append("../src/protocol/python")
-import PingMessage as Message
```
Comment (jaxxzer) [inline test/generate-json.py]: thank goodness, I was telling myself this couldn't be right, so annoying to prepend everythign

File: src/protocol/templates/ping1d.py.in [hunk 7]
```diff
@@ -113,7 +113,7 @@ class Ping1D(object):
     def handleMessage(self, msg):
         if msg.message_id in pingmessage.payloadDict:
             for attr in pingmessage.payloadDict[msg.message_id]["field_names"]:
-                setattr(self, attr, getattr(msg, attr))
```
Comment (jaxxzer) [inline src/protocol/templates/ping1d.py.in]: check the commit message

File: src/protocol/templates/pingmessage.py.in (hunk 4 above)
Comment (Williangalvani) [reply src/protocol/templates/pingmessage.py.in]: just reduced line length and made aligned indentation

File: src/protocol/templates/ping1d.py.in [hunk 8]
```diff
@@ -96,9 +97,9 @@ class Ping1D(object):
     # @param timeout: The timeout period in seconds to wait
     #
     # @return PingMessage: the message from the device if it is received within timeout period, None otherwise
-    def waitMessage(self, message_id, timeout = 0.5):
+    def wait_message(self, message_id, timeout=0.5):
```
[the remark this answers is not in the document]
Comment (Williangalvani) [reply src/protocol/templates/ping1d.py.in]: yep.

File: src/protocol/templates/pingmessage.py.in (hunk 5 above)
Comment (Williangalvani) [reply src/protocol/templates/pingmessage.py.in]: don't you think this is better?
https://github.com/bluerobotics/ping-python/pull/22/files#diff-a006c457cee7837b799e661206f71524R59

Comment (jaxxzer) [reply src/protocol/templates/pingmessage.py.in]: Yes, I'm sorry.

Comment (jaxxzer) [reply src/protocol/templates/pingmessage.py.in]: a case of me trying to process the template in my head - lazy

File: src/protocol/templates/ping1d.py.in [hunk 9]
```diff
@@ -51,8 +51,8 @@ class Ping1D(object):
             b = self.iodev.read()
 
             if self.parser.parse_byte(ord(b)) == pingmessage.PingParser.NEW_MESSAGE:
-                self.handle_message(self.parser.rxMsg)
-                return self.parser.rxMsg
+                self.handle_message(self.parser.rx_msg)
```
Comment (jaxxzer) [inline src/protocol/templates/ping1d.py.in]: how did your linter not catch this?

Comment (jaxxzer) [reply src/protocol/templates/ping1d.py.in]: or does the linter not give hits for this and it was all manual?

Comment (Williangalvani) [reply src/protocol/templates/ping1d.py.in]: It does give hints, I don't know what happened in this case

**Comment by jaxxzer** [comment]
@Williangalvani merge it when you are ready please.

**Pull request merged**